haw-pr.jenkinsfile

```diff
@@ -0,0 +1,108 @@
+        // PATH       = "${PATH}"
+                     // ":/shared/common/node-v10.15.3-linux-x64/bin"      +
+                     // ":/shared/common/java/openjdk/jdk11-x64/bin"
+        MAVEN_OPTS = '-Xmx4G'
+        JAVA_HOME  = '/usr/lib/jvm/java-11-openjdk-amd64'
+        TIMESTAMP  = new Date().format("yyyyMMddHHmm")
+   }
+
+    stages {
+        stage('Build and Test') {
+            steps {
+                echo 'Starting Xvnc'
+                wrap([$class: 'Xvnc', takeScreenshot: false, useXauthority: true]) {
+                    echo 'Building and testing, using display ' + DISPLAY
+                    // dir('n4js') {
+                        script {
+                        	def envvars = [ 
+                        		'-DDISPLAY=localhost'+DISPLAY,
+                                '-DargLine="-DDISPLAY=localhost'+DISPLAY+'"'
+                        	].join(' ')
+                            def options = [
+                                '--batch-mode',
+                                //'--quiet',
+                                '--update-snapshots',
+                                '--show-version',
+                                '-Dtycho.localArtifacts=ignore',
+                                // for adjusting the script only, renable later on:
+                                 '-Dmaven.test.failure.ignore',
+                                 "--fail-at-end",
+                                '-DWORKSPACE=' + env.WORKSPACE,
+                                '-DexcludeJRE'                               
+                            ].join(' ')
+                            def profiles = [
+                                'buildProduct',
+                                'execute-plugin-tests',
+                               // 'execute-plugin-ui-tests',
+                                'execute-ecma-tests',
+                                'execute-accesscontrol-tests',
+                                'execute-smoke-tests'
+                             //'execute-hlc-integration-tests'
+                            ].join(',')
+
+                            sh """\
+                                pwd
+                                git log -n 1
+                                npm version
+                            """
+                            sh "mvn ${envvars} clean verify -P${profiles} ${options}"
+                            // works:
+                            // sh "mvn clean verify ${options}"
+                            // for quick testing: 7
+                            // sh "mvn -o clean"
+
+                          // sh "ls -Ral builds/org.eclipse.n4js.product.build/target/repository/"
+                        }
+                    // } // end dir
+                } // end wrap
+            } // end steps
+        } // end stage
+        /*
+        stage('Deploy') {
+            steps {
+                echo 'Deploying....'
+            }
+        }*/
```

Comment:
Still necessary?

Reply:
commented out here, may be introduced again, left skeleton here -- added that comment